Dockerfile

```
@@ -14,15 +14,39 @@ RUN rm src/lib.rs
 COPY src/* src/
 # build the executable
 RUN cargo build --release
+# download jq
+RUN mkdir -p ${wd}/bin
+RUN curl -L -o /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 \
+  && chmod +x /usr/local/bin/jq
+# download cargo-local-registry (we don't use cargo install because we want it portable: we'll copy it later to the image that is run)
+RUN curl -L -o clr.tar.gz https://github.com/ChrisGreenaway/cargo-local-registry/releases/download/0.2.1/cargo-local-registry-0.2.1-x86_64-unknown-linux-musl.tar.gz \
+  && tar xvzf clr.tar.gz && chmod +x cargo-local-registry && mv cargo-local-registry /usr/local/cargo/bin
+# download and build popular crates to local registry
+RUN mkdir /local-registry
+WORKDIR /local-registry
```

I think `WORKDIR` also creates the directory.

```suggestion
WORKDIR /local-registry
```